WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -75,11 +80,20 @@ class ReaderViewModel @Inject constructor(
 
     private suspend fun restoreTabSelection(tagList: ReaderTagList) {
         withContext(bgDispatcher) {
-            appPrefsWrapper.getReaderTag()?.let {
-                val index = tagList.indexOf(it)
-                if (index != -1) {
-                    _selectTab.postValue(Event(index))
+            val index  = if (AppPrefs.getReaderTag() == null) {
```

Comment:
We might want to consider finding the correct tag in the `tagList` using `isDiscover()`.

Something like the following 
```
withContext(bgDispatcher) {
            val selectTab = { it: ReaderTag ->
                val index = tagList.indexOf(it)
                if (index != -1) {
                    _selectTab.postValue(Event(index))
                }
            }
            appPrefsWrapper.getReaderTag()?.let {
                selectTab.invoke(it)
            } ?: tagList.find { it.isDiscover }?.let {
                selectTab.invoke(it)
            }
        }
```

Wdyt?

Reply:
I'd also consider renaming the method as it doesn't just restore the selection anymore. Wdyt?

Reply:
@malinajirka 
> We might want to consider finding the correct tag in the tagList using isDiscover().

I hadn't seen isDiscover() used in the original code, so did not know it existed. Thanks for the tip.

> I'd also consider renaming the method as it doesn't just restore the selection anymore. Wdyt?  

Feels like `initializeTabSelection` may be a better name, although I feel that the method would never directly be called to set the discover tab. To further this, wdyt on wrapping the default so it can be changed in the future? 

`public boolean isDefault() {
        return isDiscover();    
    }`

Reply:
>I hadn't seen isDiscover() used in the original code, so did not know it existed. Thanks for the tip.

Yeah, that's what I assumed ;). 

>To further this, wdyt on wrapping the default so it can be changed in the future?

Love the idea! ;)

Reply:
All ready to go in ed0474d.